api/config/config.go

```diff
@@ -79,14 +79,10 @@ type BaseImageConfigs map[string]BaseImageConfig
 
 // A struct containing configuration details for each base image type
 type BaseImageConfig struct {
-	// docker image name with path
-	ImageName string `json:"imageName"`
-
-	// Dockerfile Path within the build context
-	DockerfilePath string `json:"dockerfilePath"`
-
-	// GCS URL Containing build context
-	BuildContextURI string `json:"buildContextURI"`
+	ImageName       string `json:"imageName"`       // docker image name with path
```

Comment:
nit: The convention in Golang is to put comments on top of the field so that godoc can generate the documentation automatically. You can read more about writing more idiomatic golang in https://go.dev/doc/effective_go . Highly recommended if you haven't read it before.

Reply:
oh ok will change back. I shifted it to side because it could create nice formatting.